sdks/java/core/src/main/java/org/apache/beam/sdk/options/SdkHarnessOptions.java

```diff
@@ -427,4 +427,16 @@ public Duration create(PipelineOptions options) {
           : Duration.ofMinutes(1);
     }
   }
+
+  /**
+   * The time limit (in minute) that an SDK worker allows for a PTransform operation before
+   * signaling the runner harness to restart the SDK worker.
```

Comment:
nit: clarify that zero means no time-limit

"There is no time limit if the value is zero."